noir-projects/aztec-nr/aztec/src/oracle/get_membership_witness.nr

```diff
@@ -2,6 +2,7 @@ use dep::protocol_types::{constants::{ARCHIVE_HEIGHT, CONTRACT_TREE_HEIGHT, NOTE
 
 global CONTRACT_TREE_ID = 0;
 global NOTE_HASH_TREE_ID = 2;
+global L1_TO_L2_MESSAGE_TREE = 4;
```

Comment:
```suggestion
global L1_TO_L2_MESSAGE_TREE_ID = 4;
```

To keep it consistent.

Reply:
Think we can actually delete this one, since it did not ended up being used. Or we should have all the trees instead 🤔